code/controllers/subsystem/blackbox.dm

```diff
@@ -35,7 +35,7 @@ SUBSYSTEM_DEF(blackbox)
 		if(M.client)
 			playercount += 1
 	var/admincount = GLOB.admins.len
-	var/datum/DBQuery/query_record_playercount = SSdbcore.NewQuery("INSERT INTO [format_table_name("legacy_population")] (playercount, admincount, time, server_ip, server_port) VALUES ([playercount], [admincount], '[SQLtime()]', INET_ATON(IF('[world.internet_address]' LIKE '', '0', '[world.internet_address]')), '[world.port]')")
+	var/datum/DBQuery/query_record_playercount = SSdbcore.NewQuery("INSERT INTO [format_table_name("legacy_population")] (playercount, admincount, time, server_ip, server_port, round_id) VALUES ([playercount], [admincount], '[SQLtime()]', INET_ATON(IF('[world.internet_address]' LIKE '', '0', '[world.internet_address]')), '[world.port]', '[GLOB.round_id]')")

```

Comment:
Why is the value quoted?

Reply:
I copy-pasted it from somewhere, it makes essentially no difference to us for an `int` type.

Reply:
Normally, doing that for a number field would be a sql error.

However we run mysql under some laxer settings.

One day we can strive for a sql_strict compatible sql base.

Reply:
I'm kinda curious to how heavy a load you'd have to try and run before seeing a performance difference when forcing the value to be cast before insert, but not curious enough to benchmark it.